packages/atti-components/src/components/Slider/styles/SliderValue.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+import Label from "../../Typographies/Label";
+
+const SliderValue = styled(Label)`
+  position: absolute;
+  ${({theme: {components: {slider: {sliderValue: {backgroundColor, bottom, color, height, width}}}}}) => `
+    background-color: ${backgroundColor};
+    bottom: ${bottom};
+    color: ${color};
+    height: ${height};
+    line-height: ${height};
+    width: ${width};
+  `}
+  border-radius: 100%;
+  text-align: center;
+  animation: sliderValueAnimation ease-in 0.2s;
```

Comment:
Could use a theme value for the animation time

Reply:
Done